packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3725,6 +3841,9 @@ export class ContainerRuntime
 				this.outbox.submitIdAllocation(message);
 			} else {
 				this.submitIdAllocationOpIfNeeded();
+				this.documentsSchemaController.onMessageSent(
+					(content: IDocumentSchemaChangeMessage) => this.outbox.submit(message),
```

Comment:
I think this needs a comment here, it took me a bit to figure out what's going on.